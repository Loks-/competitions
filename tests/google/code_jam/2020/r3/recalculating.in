7
2 4
0 0
5 0
2 1
0 0
5 0
2 4
0 0
4 4
2 4
0 0
5 1
3 4
0 0
1 1
2 3
2 2
0 0
0 1
2 2
0 0
4 4
