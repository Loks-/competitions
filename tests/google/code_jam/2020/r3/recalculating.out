Case #1: 27 119
Case #2: 0 1
Case #3: 0 1
Case #4: 1 5
Case #5: 101 109
Case #6: 19 23
Case #7: 0 1
